<?xml version="1.0"?>
<launch>
  <arg name="ns" default="myrobot"/>
  <arg name="id_subsystem" default="133"/>
  <arg name="id_node" default="64"/>
  <group ns="$(arg ns)">
      <node name="iop_illumination_driver" pkg="fkie_iop_component" type="iop_component">
        <param name="capability_group" value="iop_bridge"/>
        <rosparam param="services">
          [
            iop_poewr_plant_fkie: "PowerPlantManager",
            fkie_iop_discovery: "DiscoveryClient",
            fkie_iop_accesscontrol: "AccessControl",
            fkie_iop_events: "Events",
            fkie_iop_events: "EventsClient",
            fkie_iop_transport: "Transport",
          ]
        </rosparam>
        <param name="iop_address" value="$(arg id_subsystem).$(arg id_node).63"/>
        <!-- ON, OFF, 0: not supported, 1: supported (unknown state) -->
        <rosparam subst_value="true">
          PowerPlantManager:
            power_plants:
              - "1":
                - battery:
                  - voltage: 58
        </rosparam>
      </node>
  </group>
</launch>